### DISCUSSION TITLE
lasso eraser

### DISCUSSION THREAD

**Discussion by gonewiththewind0726** [xournalpp/xournalpp#5278] [Ideas] [answered]
sir your creation help me a lot in my mathclass ，my English is poor so the follwing words istranslated by Google ，recently I use the eraser and find the eraser offen erase other approaching strokes when I use it，I hope you may add lasso eraser like notein or adobe photoshop to help precisely erase instead of rubbing approaching white

**Comment by rolandlo** [comment, marked as answer]
If you just want to delete whole strokes, you could use the lasso selection and delete the whole selection.
